frontend/src/metabase/entities/containers/EntityListLoader.jsx

```diff
@@ -9,27 +9,44 @@ import entityType from "./EntityType";
 import paginationState from "metabase/hoc/PaginationState";
 import LoadingAndErrorWrapper from "metabase/components/LoadingAndErrorWrapper";
 
-export type Props = {
-  entityType?: string,
-  entityQuery?: ?{ [key: string]: any },
-  reload?: boolean,
-  wrapped?: boolean,
-  debounced?: boolean,
-  loadingAndErrorWrapper: boolean,
-  selectorName?: string,
-  children: (props: RenderProps) => ?React.Element,
+const propTypes = {
+  entityType: PropTypes.string,
+  entityQuery: PropTypes.oneOfType([PropTypes.func, PropTypes.object]),
+  reload: PropTypes.bool,
+  wrapped: PropTypes.bool,
+  debounced: PropTypes.bool,
+  loadingAndErrorWrapper: PropTypes.bool,
+  keepPreviousList: PropTypes.bool,
+  selectorName: PropTypes.string,
+  children: PropTypes.func,
+
+  // via entityType HOC
+  entityDef: PropTypes.object,
+
+  // via react-redux connect
+  list: PropTypes.arrayOf(PropTypes.object),
+  metadata: PropTypes.object,
+  loading: PropTypes.bool,
+  loaded: PropTypes.bool,
+  fetched: PropTypes.bool,
+  error: PropTypes.any,
+  allLoading: PropTypes.bool,
+  allLoaded: PropTypes.bool,
+  allFetched: PropTypes.bool,
+  allError: PropTypes.bool,
+  dispatch: PropTypes.func,
 };
 
-export type RenderProps = {
```

Comment:
I totally love removing flow types as we touch files.
Let's see how our teammates react to the initiative.